src/index.html

```diff
@@ -7,6 +7,45 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+
+    <div
+    class="
+    calendar
+    calendar--start-day-wed
+    calendar--mouth-lenght-29
+    "
+    >
```

Comment:
```suggestion
    <div class="calendar calendar--start-day-wed calendar--mouth-length-29">
```
the length should be less than 80 characters in one row

Also, typo not ~lenght~ but **length**